src/plugins/es_ui_shared/static/forms/helpers/field_validators/min_length.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ValidationFunc, ValidationError } from '../../hook_form_lib';
+import { hasMinLengthString } from '../../../validators/string';
+import { hasMinLengthArray } from '../../../validators/array';
+import { ERROR_CODE } from './types';
+
+export const minLengthField = ({
```

Comment:
@sebelga I see this and a few other validators aren't in use in the Index Templates UI. Do you intend to merge these along with the others? Are we going to test them by consuming them in a UI?

Reply:
So the helpers of the PR have to be seen as a library and **not code running**. The same that eui exposes components and some app will use 2 other 20. This is the reason there are more validators than the ones currently being used in Template UI (I created this PR with 2 unrelated changes to communicate the value of the helpers, not to show every single one of them in use).

For the tests, yes all the validators will have their tests. I will add them in a separate PR.